ui/src/app/pages/workspace/workspace-about.tsx

```diff
@@ -179,28 +188,28 @@ const WorkspaceInfoTooltipText = () => {
 
 export const WorkspaceAbout = fp.flow(
   withUserProfile(),
-  withCdrVersions()
+  withCdrVersions(),
+  withCurrentWorkspace()
```

Comment:
The component was already using store, so rather than maintaining state, this seemed simpler.